src/client.rs

```diff
@@ -442,8 +443,9 @@ impl Client {
         steam_app_id: usize,
     ) -> Result<GameInfo, Box<dyn std::error::Error>> {
         let url = get_game_by_steam_app_id_url(self.base_url.as_str(), steam_app_id);
-        let response = self.make_request::<GameInfo>(url.as_str()).await?;
-        Ok(response)
+        let response = self.make_request::<Response<GameInfo>>(url.as_str()).await?;
+        let game_info = response.data.unwrap();
```

Comment:
Instead of unwrapping, use the ? to return an error instead of a panic
```suggestion
        let game_info = response.data?;
```